lib/event_sourcery/postgres/event_store.rb

```diff
@@ -33,22 +33,16 @@ def sink(event_or_events, expected_version: nil)
 
       def get_next_from(id, event_types: nil, limit: 1000)
         query = events_table.
-          order(:id).
-          where(Sequel.lit('id >= ?', id)).
-          limit(limit)
-        if event_types
-          query = query.where(type: event_types)
-        end
-        query.map do |event_row|
-          build_event(event_row)
-        end
+                order(:id).
+                where(Sequel.lit('id >= ?', id)).
+                limit(limit)
+        query = query.where(type: event_types) if event_types
+        query.map { |event_row| build_event(event_row) }
```

Comment:
humm yeah could be a little confusing , maybe just leave it as the inline block for now.